### ISSUE TITLE
python2: astropy tables with unicode don't print

### ISSUE THREAD

**Issue by keflavich** [astropy/astroquery#689]
Example:

```
from astroquery.alma import Alma
from astropy import units as u
Alma.query_region(coordinate=SkyCoord('04:54:08.85 -03:00:28.8', unit=('hour','deg'), frame='icrs'), radius=1*u.arcmin)
```

results in:

```
Out[14]: Traceback (most recent call last):
  File "/Users/adam/anaconda/envs/astropy27/lib/python2.7/site-packages/IPython/core/formatters.py", line 697, in __call__
    printer.pretty(obj)
  File "/Users/adam/anaconda/envs/astropy27/lib/python2.7/site-packages/IPython/lib/pretty.py", line 383, in pretty
    return _default_pprint(obj, self, cycle)
  File "/Users/adam/anaconda/envs/astropy27/lib/python2.7/site-packages/IPython/lib/pretty.py", line 503, in _default_pprint
    _repr_pprint(obj, p, cycle)
  File "/Users/adam/anaconda/envs/astropy27/lib/python2.7/site-packages/IPython/lib/pretty.py", line 685, in _repr_pprint
    output = repr(obj)
  File "/Users/adam/repos/astropy/astropy/table/table.py", line 812, in __repr__
    return self._base_repr_(html=False, max_width=None)
  File "/Users/adam/repos/astropy/astropy/table/table.py", line 799, in _base_repr_
    max_lines=max_lines, tableclass=tableclass)
  File "/Users/adam/repos/astropy/astropy/table/pprint.py", line 477, in _pformat_table
    align=align_)
  File "/Users/adam/repos/astropy/astropy/table/pprint.py", line 223, in _pformat_col
    col_strs = list(col_strs_iter)
  File "/Users/adam/repos/astropy/astropy/table/pprint.py", line 391, in _pformat_col_iter
    col_str = format_func(col_format, col[i])
  File "/Users/adam/repos/astropy/astropy/table/pprint.py", line 28, in <lambda>
    _format_funcs = {None: lambda format_, val: text_type(val)}
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 52: ordinal not in range(128)
```

Specific problem:
```
In [18]: query['Project abstract'].data
Out[18]:
masked_BaseColumn(data = [ 'We propose to conduct deep ALMA observation of two z\xe2\x88\xbc6 sub-L\xe2\x88\x97 star-forming galaxies to detect their [CII] emission line and derive from the line flux their star formation activity. These two Lyman-break galaxies are unique multiple image systems found in the massive cluster lenses Abell 383 and MS0451-03 at redshift 6.027 and 6.703 respectively. Their lensing magnification is ~11 and 100 respectively thus allowing to probe the properties of &#34;normal&#34; low luminosity galaxies at early times in the Universe. The proposed observation will lead to a secure [CII] detection. However, would we not detect [CII] the upper limit we can derived will have strong consequences on our understanding of star formation in the early Universe advocating for unexpected strong evolution in the galaxy formation models.'
 'We propose to conduct deep ALMA observation of two z\xe2\x88\xbc6 sub-L\xe2\x88\x97 star-forming galaxies to detect their [CII] emission line and derive from the line flux their star formation activity. These two Lyman-break galaxies are unique multiple image systems found in the massive cluster lenses Abell 383 and MS0451-03 at redshift 6.027 and 6.703 respectively. Their lensing magnification is ~11 and 100 respectively thus allowing to probe the properties of &#34;normal&#34; low luminosity galaxies at early times in the Universe. The proposed observation will lead to a secure [CII] detection. However, would we not detect [CII] the upper limit we can derived will have strong consequences on our understanding of star formation in the early Universe advocating for unexpected strong evolution in the galaxy formation models.'
 'We propose to conduct deep ALMA observation of two z\xe2\x88\xbc6 sub-L\xe2\x88\x97 star-forming galaxies to detect their [CII] emission line and derive from the line flux their star formation activity. These two Lyman-break galaxies are unique multiple image systems found in the massive cluster lenses Abell 383 and MS0451-03 at redshift 6.027 and 6.703 respectively. Their lensing magnification is ~11 and 100 respectively thus allowing to probe the properties of &#34;normal&#34; low luminosity galaxies at early times in the Universe. The proposed observation will lead to a secure [CII] detection. However, would we not detect [CII] the upper limit we can derived will have strong consequences on our understanding of star formation in the early Universe advocating for unexpected strong evolution in the galaxy formation models.'],
                  mask = [False False False],
            fill_value = N/A)

```

EDIT: note the dtype of this column is `('Project abstract', 'S4000')`

This is therefore a problem with astropy.table's pprint function

**Comment by bsipocz** [comment]
If it's ``astropy.table`` problem, could you open an issue there, too? Maybe it's an easy fix for 1.2.

**Comment by keflavich** [comment]
Yeah, it turns out this is a generic astropy tables problem; I encountered it in an entirely related (...not even astro...) project yesterday.

**Comment by pllim** [comment]
Potential fix in astropy/astropy#4946

**Comment by bsipocz** [comment]
Fixed in astropy.
